apps/landing/components/Terms/Content.tsx

```diff
@@ -0,0 +1,291 @@
+// TODO: convert this component to server component once the Typography task is done
+"use client";
+
+import React from "react";
+import cn from "classnames";
+import { useFormatMessage } from "@/hooks/intl";
+import { Typography } from "@litespace/ui/Typography";
+
+import Facebook from "@litespace/assets/Facebook";
+import Youtube from "@litespace/assets/Youtube";
+import Insta from "@litespace/assets/Instagram";
+import Linkedin from "@litespace/assets/Linkedin";
+import Whatsapp from "@litespace/assets/Whatsapp";
+import { SocialLink } from "@/components/common/SocialLink";
+import { BulletList } from "@/components/common/BulletList";
+import { NumberedList } from "@/components/common/NumberedList";
+
+const socials = [
+  {
+    href: "https://www.instagram.com/litespace_eng",
+    icon: <Insta className="w-8 h-8 fill-brand-500" />,
+  },
+  {
+    href: "https://www.facebook.com/litespace.eng",
+    icon: <Facebook className="w-8 h-8 fill-brand-500" />,
+  },
+  {
+    href: "https://www.linkedin.com/company/litespace-eng",
+    icon: <Linkedin className="w-8 h-8 fill-brand-500" />,
+  },
+  {
+    href: "https://wa.me/201503310108",
+    icon: <Whatsapp className="w-8 h-8 fill-brand-500" />,
+  },
+  {
+    href: "https://www.youtube.com/@litespace_en",
+    icon: <Youtube className="w-8 h-8 fill-brand-500" />,
+  },
+];
```

Comment:
The links are well is also shared between the terms and the privacy page. It should be moved into a shared component as well. 

All of this should be one shared component. 

![image](https://github.com/user-attachments/assets/a49ada5d-4c8e-4750-a79b-0e0b72642b83)